### PR TITLE
Modify tasks

### PR DESCRIPTION
**Pull request by HopeNadyaHope**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,97 @@
-The following tasks must be completed:
+# Tasks
 
-1. Write a method that takes an array of String elements as an input and returns Map <String, Integer>, where String is the value from the array and Integer is the number of entries in the array.
-2. Write an Iterator over an array.
-3. Write a method to build a frequency dictionary of the letters of the English alphabet.
+## 1. Lessons Getter
+
+Implement the program which gets timetable and returns `set` of all lessons without duplicates. Timetable is
+a `Map<String, List<String, String>>`, where the `key` - the day of the week and the `value` - `List<String> `of lessons
```
Comment (egoist301) [inline README.md]: Could you remove third word `String`? 😄
`Map<String, List<String>>`

File: README.md [hunk 2]
```diff
@@ -1,5 +1,97 @@
...
+
+**Output**
+
+```
+English, Mathematics, Chemistry, Literature, Physics
+```
+
+## 2. Map from keys Creator
+
+Implement the program which gets `Map<String, Integer>` as parameter and returns a new `Map<Integer, Set<String>>`,
```
Comment (egoist301) [inline README.md:36]: I don't understand why you use `Map<String, Integer>` as parameter if you don't use values from map.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,97 @@
...
+
+```
+3 - one, two, ten
+4 - five
+5 - three
+```
+
+## 3. Developer Project Finder
+
+Implement the program which gets `Map<String, List<String, String>` of projects and its developers and the required
```
Comment (egoist301) [inline README.md]: The same, `Map<String, List<String>>`.

File: src/test/java/com/epam/mjc/collections/combined/DeveloperProjectFinderTest.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+                        List.of()),
+
+                Arguments.of("HaveProjects",
+                        Map.of("AXE", Set.of("Helen", "Bob", "Marian", "Tom"),
+                                "MazPay", Set.of("James", "John", "Robert"),
+                                "SeaV", Set.of("John", "Robert", "Angelina", "David"),
+                                "DelFo", Set.of("Patricia", "Jessica", "Mark", "Paul"),
+                                "ReaNon", Set.of("George", "Kyle", "Robert", "Brandon", "Austin")),
+                        "Robert",
+                        List.of("ReaNon", "MazPay", "SeaV"))
```
Comment (egoist301) [inline src/test/java/com/epam/mjc/collections/combined/DeveloperProjectFinderTest.java:38]: Is it correct order?

File: src/test/java/com/epam/mjc/collections/combined/LessonsGetterTest.java [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+                Arguments.of("TimetableWithoutLessons",
+                        Map.of("Monday", List.of(),
+                                "Tuesday", List.of(),
+                                "Wednesday", List.of(),
+                                "Thursday", List.of(),
+                                "Friday", List.of()),
+                        Set.of()),
+
+                Arguments.of("TimetableWithoutLessons",
```
Comment (egoist301) [inline src/test/java/com/epam/mjc/collections/combined/LessonsGetterTest.java]: `TimetableWithLessons` maybe?

File: README.md (hunk 2 above)
Comment (HopeNadyaHope) [reply README.md:36]: the idea was to use `keySet()`

File: src/test/java/com/epam/mjc/collections/combined/DeveloperProjectFinderTest.java (hunk 4 above)
Comment (HopeNadyaHope) [reply src/test/java/com/epam/mjc/collections/combined/DeveloperProjectFinderTest.java:38]: Yes, _projects for requested developer in **descending** order of project
name length. If the names of the projects are of the same length, then sort **alphabetically in reverse order**_

**Review by egoist301 — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -65,7 +65,7 @@ ten - 10
 
 ## 3. Developer Project Finder
 
-Implement the program which gets `Map<String, List<String, String>` of projects and its developers and the required
+Implement the program which gets `Map<String, List<String>` of projects and its developers and the required
```
Comment (egoist301) [inline README.md]: `Map<String, List<String>>`

**Pull request merged**